playbooks/tests/tasks/integration.yml

```diff
@@ -48,6 +30,13 @@
   retries: 6
   delay: 10
 
+- name: ensure neutron meta-data is running
+  shell: . /root/stackrc; ps aux | grep neutron-ns-metadata-proxy | grep $(neutron router-show default | grep " id " | awk '{ print $4 }')
```

Comment:
Minor thing for the future: you should use `pgrep <whatever>` (here, use `pgrep -f -a` for matching and outputting the whole line) rather than `ps | grep`